site/src/.scaffold/standard.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<meta http-equiv="X-UA-Compatible" content="ie=edge" />
+		<title>{{title}} | Atlas Design | Microsoft</title>
+		<meta property="og:title" content="{{title}} | Atlas Design | Microsoft" />
+		<meta property="og:type" content="website" />
+		<meta name="color-scheme" content="light dark" />
+		<link rel="stylesheet" href="localhost:8888/index.css" />
```

Comment:
I bet if you add `@microsoft/atlas-css` to the package.json as a dependency you'll be able to use `href="@microsoft/atlas-css"` here.

Reply:
was not immediately successful there, I'll address the inclusion of styles in another PR